.github/workflows/pr.yml

```diff
@@ -1,5 +1,26 @@
 name: CI
-on: [push]
+
+on:
+  push:
+    branches:
+      - main
+      - staging
+      - release/*
+  pull_request:
+    branches:
+      - main
+      - staging
+      - release/*
```

Comment:
Are maintaining release/* branches for sample components?

Reply:
I don't know if I understand, but I've added those branches in case if someone want to add some bug fixes to old releases. I hope that it reduce number of duplicated jobs (push and pull request) but I'm not sure about that.